parlai/crowdsourcing/tasks/qa_data_collection/task_description.html

```diff
@@ -0,0 +1,17 @@
+<h1>If you see this, it means the task description has loaded successfully</h1>
+<br />
+<h3>What is this task?</h3>
+<p>
+  In this task, you'll take 2 turns (or perhaps more turns as specified by the
```

Comment:
Hmm - I'd probably remove the "(or perhaps more turns" line, because more than 2 turns doesn't appear to be supported by the code at the moment